src/main/g8/src/main/scala/$package$/$name__Camel$.scala

```diff
@@ -3,7 +3,7 @@ package $package$
 import cats.effect.IOApp
 import cats.effect.IO
 
-object Main extends IOApp.Simple {
+object $name;format="Camel"$ extends IOApp.Simple {
```

Comment:
I wouldn't change this file. Personally I use this pattern with `Main` as my main class 100% of the time (and I've seen that pattern in prod in both my recent jobs). The package name is enough to disambiguate.

Reply:
Seems legit